frontend/packages/dev-console/integration-tests/features/web-terminal/web-Terminal-adminuser.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Web Terminal for Developer user
+    As a user with admin rights, I should be able to use web terminal
```

Comment:
Feature and statement do not match. If it is for the user with admin rights then the feature should be for the Admin user

Reply:
Updated